### PR TITLE
[Bluetooth] Implemented bluetooth enable, create id, delete id

### PR DESCRIPTION
**Pull request by amitpurwar**
[Bluetooth] Implemented bluetooth enable, create id, delete id

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [d88583e5238680499ea62f664de1314bd68a41f2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [d88583e5238680499ea62f664de1314bd68a41f2] - Code Rule Check Result:
os/include/tinyara/bluetooth/bluetooth.h:171: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/bluetooth/bluetooth.h:171: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/tinyara/bluetooth/bluetooth.h:172: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/bluetooth/bluetooth.h:172: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/tinyara/bluetooth/bluetooth.h:173: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/bluetooth/bluetooth.h:173: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/tinyara/bluetooth/bluetooth.h:174: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/bluetooth/bluetooth.h:174: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/tinyara/bluetooth/bluetooth.h:175: ERROR: [IDT_M_TAB] code indent should use tabs where possible
os/include/tinyara/bluetooth/bluetooth.h:175: ERROR: [IDT_M_TAB] please, no spaces at the start of a line
os/include/tinyara/bluetooth/bt_hci.h:58: ERROR: [SPC_M_SEP] space required after that close brace '}'
os/include/tinyara/bluetooth/bt_hci.h:59: ERROR: [SPC_M_SEP] space required after that close brace '}'

**Comment by Wootak** [comment]
Dear Amit,
Please apply tizen coding rule.
I sent the script to fix code.
Thank you.

File: os/include/tinyara/bluetooth/bt_hci.h [hunk 1]
```diff
@@ -172,6 +172,52 @@
 #define BT_HCI_EVT_LE_ADVERTISING_REPORT      0x02
 #define BT_HCI_EVT_LE_LTK_REQUEST             0x05
 
+#define CONFIG_BT_DEVICE_NAME_MAX		65
+#define CONFIG_BT_DEVICE_NAME			"TizenRT Bluetooth"
```
Comment (dh79pyun) [inline os/include/tinyara/bluetooth/bt_hci.h]: Please use the default device name as "TizenRT"

**Comment by seinfra** [comment]
Target : [f4f15205e93aba7b2db03d7456c89f2f19d03dbf] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f4f15205e93aba7b2db03d7456c89f2f19d03dbf] - Code Rule Check Result:
os/include/tinyara/bluetooth/bt_hci.h:58: ERROR: [SPC_M_SEP] space required after that close brace '}'
os/include/tinyara/bluetooth/bt_hci.h:59: ERROR: [SPC_M_SEP] space required after that close brace '}'

**Comment by seinfra** [comment]
Target : [312c7e936bff22a8d7c5cb29b159afff97ca0174] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [312c7e936bff22a8d7c5cb29b159afff97ca0174] - Code Rule Check Result:
os/include/tinyara/bluetooth/bt_hci.h:58: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'
os/include/tinyara/bluetooth/bt_hci.h:59: ERROR: [SPC_M_SEP] space prohibited before that close parenthesis ')'

**Comment by seinfra** [comment]
Target : [b7d9e9b53f4308bd68ef6e48113944854fbc3942] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [b7d9e9b53f4308bd68ef6e48113944854fbc3942] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [9928d285987a7292ee7f0d27d7eda09a42874797] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9928d285987a7292ee7f0d27d7eda09a42874797] - Code Rule Check OK.

**Comment by sunghan-chang** [comment]
@amitpurwar Please check our commit rules in wiki page. This is not acceptable.

**Pull request merged**